src/functional

```diff
@@ -32,15 +50,14 @@ foreach ($finder as $testSuite) {
     $temp = new Temp($testSuite->getBasename());
     $temp->initRunFolder();
 
-    $fs = new Filesystem();
     $fs->mirror($testSuite->getPathname() . '/source/data', $temp->getTmpFolder());
 
     $fs->mkdir($temp->getTmpFolder() . '/in/tables', 0777);
     $fs->mkdir($temp->getTmpFolder() . '/in/files', 0777);
     $fs->mkdir($temp->getTmpFolder() . '/out/tables', 0777);
     $fs->mkdir($temp->getTmpFolder() . '/out/files', 0777);
 
-    $runCommand = "KBC_DATADIR={$temp->getTmpFolder()} php /code/src/run.php";
+    $runCommand = "KBC_DATADIR={$temp->getTmpFolder()} php " . $script;
```

Comment:
potenciálně tady jde spustit libovolný php skript nebo cokoli jiného (např. `-v; cat /etc/shadow`), ale nemyslím, že je to problém, když to běží v kontejneru, nebo ano?

Reply:
potencionalne mi prijde, ze je to uplne jedno, kdyz je to clicko, tak si muzes spustit cat /etc/shadow rovnou ne?